static/app/components/performance/waterfall/miniHeader.tsx

```diff
@@ -8,26 +8,24 @@ export const DividerSpacer = styled('div')`
 const MINI_HEADER_HEIGHT = 20;
 
 export const ScrollbarContainer = styled('div')`
-  display: flex;
-  align-items: center;
+  display: block;
   width: 100%;
-  height: ${MINI_HEADER_HEIGHT}px;
-  left: 0;
-  bottom: 0;
+  height: ${MINI_HEADER_HEIGHT + 50}px;
```

Comment:
@Zylphrex Since this container is scrollable on the horizontal axis (from `overflow-x: scroll`); the browser will add a horizontal scroll bar (which may or may not have a scrollbar). This component's parent will have `overflow: hidden`, and by extending this component's height, we can hide the horizontal scrollbar. This is the same height extension I used in https://github.com/getsentry/sentry/pull/22504